### PR TITLE
Add additional info(Metadata and Metrics) for codepush deployment history

### PR DESCRIPTION
**Pull request by alexandergoncharov**
**Issue:** Old `code-push` cli has install metrics with `deployment history` command. `Appcenter-cli` doesn't have metrics in `deployment history` command.

**Solution:** Added Metrics for output of `deployment history` command and made output view as in code-push cli.

**Before:**
image

**After:**
image

Fix implemented by @yuri-kulikov.

### CODE REVIEW

File: src/commands/codepush/deployment/history.ts [hunk 1]
```diff
@@ -45,4 +73,56 @@ export default class CodePushDeploymentHistoryCommand extends AppCommand {
...
     }
   }
+
+  private generateReleaseMetricsString(release: models.CodePushRelease, metrics: models.CodePushReleaseMetric[], releasesTotalActive: number): string {
+    let metricsString: string = "";
+
+    const releaseMetrics: models.CodePushReleaseMetric = metrics.find((metric) => metric.label === release.label);
+    if (releaseMetrics) {
+
+      const activePercent: number = (releaseMetrics.downloaded) ? releaseMetrics.active / releasesTotalActive * 100 : 0.0;
```
[the remark this answers is not in the document]
Comment (yuri-kulikov) [reply src/commands/codepush/deployment/history.ts]: Oh, sure, I think in that case it's better to use this: 
```TypeScript
(releasesTotalActive) ? releaseMetrics.active / releasesTotalActive * 100 : 0.0;
```

**Comment by yuri-kulikov** [comment]
Yeah, we are working on tests right now but it’s just a separate user story

File: src/commands/codepush/deployment/history.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (owenniblock) [reply src/commands/codepush/deployment/history.ts]: That works for me.

**Comment by owenniblock** [comment]
No problem - ping me when you're happy with the tests and I'll take a fresh look.

**Comment by yuri-kulikov** [comment]
Hey @owenniblock, thanks for reply!
Is it possible if we’ll add tests in a separate PR and will merge this one if you don’t have any objections?
Otherwise we can add tests in this PR, just worried a little bit if PR size wouldn’t be too large and thus harder to review.
Anyway we’re flexible with whatever discussion you prefer, just let us know what works best for you (await merging this one and push changes to tests directly here or do it in a separate PR)?

**Comment by owenniblock** [comment]
@yuri-kulikov - I'd rather have the tests in this PR if that's OK? If I merge this and then the tests highlight some issues we hadn't of thought of - it will make the history a little messy. Having the tests should help the review (since it will show that areas of code behave as we'd expect).

Also if I merge to master we risk releasing before the tests are ready. :)

**Comment by yuri-kulikov** [comment]
@owenniblock, thanks, let’s stick with this approach, I’ll ping you when we push changes for tests.

File: test/commands/codepush/deployment/history-test.ts [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+    "Active: 25% (2 of 8)" + lineFeed + "Installed: 4",
+    "No installs recorded" + lineFeed + "Disabled: Yes",
+    "No installs recorded" + lineFeed + "Rollout: 0%",
+    "Active: 13% (1 of 8)" + lineFeed + "Installed: 1" + lineFeed + "Rollout: 56%",
+    "Active: 0% (0 of 8)" + lineFeed + "Disabled: Yes",
+    "Active: 0% (0 of 8)",
+    "No installs recorded" + lineFeed + "Rollout: 0%",
+  ];
+
+  const fakeReleasesResponse = [
```
Comment (AndreyMitsyk) [inline test/commands/codepush/deployment/history-test.ts]: Could you please extract test data like this to separate file? It does not looks good when we have ~120 lines of test data before actual tests.

Comment (yuri-kulikov) [reply test/commands/codepush/deployment/history-test.ts]: Yes, sure, I'll do it.

**Comment by yuri-kulikov** [comment]
Test data extracted to separate file.

**Pull request merged**